_Sidebar.md

```diff
@@ -120,6 +120,7 @@
   * [[Build process|Build-process]]
   * [[Team structure|Team-Structure]]
   * [[Triaging Process|Triaging-process]]
+  * [[Post Installation Actions|Post-Installation-Actions]]
```

Comment:
This needs to be placed somewhere more discoverable, perhaps as a sub-bullet point under "Installing Oppia"

Or alternatively, you could add a new bullet point called "Getting started with the codebase" under "Installing Oppia" and we could have this page as a sub-bullet point of it. We could also move the LaCE and CD onboarding guides up here to this section too -- I feel like they are currently not very discoverable. Also the "Tour of Oppia webpages."

What do you think?

Reply:
Hi @seanlip, I think this can be our flow...
* Installing oppia
  * Post Installation Actions
  * ( maybe tutorials or something )
  * Team onboarding guides
    * LaCE Team
    * CD Team
   
 I am not sure about the "Tour of Oppia webpages" It is already within the "codebase overview" section, and it is quite at the right place.

Reply:
Hm, I htink the issue with this layout is that the items under "Installing Oppia" aren't really anything to do with installing Oppia.

How about something like this:

- Installing Oppia
- Getting started with the codebase
  - Populating data on your local server
  - How to access different Oppia webpages
  - Tutorials
     - How to make a frontend change   <-- Ashwath's tutorial
     - (etc.)
  - Team-specific onboarding guides
    - LaCE team
    - CD team

Here, "Populating data on your local server" is a rename of "Post-Installation Actions", and "How to access different Oppia webpages" is a rename of "Tour of Oppia webpages". The idea here is to give devs things they can do immediately after installation and as they begin their first issue. Currently, "Tour of Oppia webpages" is buried in a longer menu and I don't think folks actually discover it.

What do you think about the above?

Reply:
@seanlip, Have made required changes. PTAL.